plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -109,16 +148,17 @@ export const TechDocsReaderPageHeader = (
 
   return (
     <Header
-      title={siteName ? siteName : '.'}
-      pageTitleOverride={siteName || name}
-      subtitle={
-        siteDescription && siteDescription !== 'None' ? siteDescription : ''
-      }
-      type="Docs"
+      type="Documentation"
       typeLink={docsRootLink}
+      title={title || skeleton}
+      subtitle={subtitle || skeleton}
     >
+      <Helmet titleTemplate="%s">
+        <title>{tabTitle}</title>
+      </Helmet>
```

Comment:
i think the helmet is meant to go around the entire contents, so that you can nest helmets inside each other